modules/ROOT/pages/rate-limiting.adoc

```diff
@@ -0,0 +1,237 @@
+
+You can configure the policy for multiple groups of requests by using identifiers in the policy configuration. Each group has a separate available quota for its window. 
+
+To understand how the Rate Limiting policy works, consider an example in which the configuration of 3 requests every 10 seconds allows or restricts incoming requests, based on the quota available in that window:
+
+image::rate-limiting-generic-example.png[Rate Limiting Policy Example,70%,85%]
+
+In the first window, because the quota is reached with the third request, all subsequent requests are rejected until the window closes. In the second window, only two of the three requests are processed and the quota remaining is dropped after the window time has elapsed.
+
+An accepted request passes through the API to the backend. Conversely, a rejected request displays 1429 status for HTTP (or either 400 or 500 if the API is WSDL) and does not reach the backend.
+
+A rejected request on the other hand, displays a “429 status for HTTP, ” (or either 400 or 500 if the API is WSDL) and does not reach the backend.
+
+== Examples
+
+Consider how the previously described configuration of 3 requests every 10 seconds works when the configuration is `clusterized` and `identifiers` are added.
+
+=== Configuring Identifiers by Using Regular Strings
+
+When you use the UI to add `identifiers` to the policy configuration, you can define groups of requests. The configured limits apply independently to each group. You can also use the `identifier` `#[attributes.method]` for one bucket per HTTP method in your Rate Limiting policy configuration. 
+
+//If you can split the universe of possible requests into a finite number of groups, you can assign each group with a selector key (String). One Rate Limit algorithm is created per key.  
+
+An `identifier` is a non-obligatory parameter.  By default, the `identifier` has no value.
+Based on whether you accept the default or provide a value, the policy performs in the following ways:
+
+* Not configured (default)
++
+The Rate Limiting quota applies to every request per bucket or group.
+* Configured for an obligatory HTTP header
++
+Each header has its own quota. Header values are case-sensitive. Quotas are created using the _lazy creation_ strategy.
+* Configured for a non-obligatory HTTP header
++
+Custom header, payload, query parameter, or expression values each have their own quotas.  
++
+The `identifier`, if not sent in the request, defaults to an empty value, having its own quota. This behavior allows the Rate Limiting policy to be applied to an API consumed by uncontrolled clients, and at the same time accommodates special buckets for the clients sending the `identifier` (which might be internal controlled services).
+//ask Mariano what does the braces refer to? What might be internal controlled services
```

Comment:
I try to refer to the fact that you could say "ok I've created a special bucket just foryou, please you identifier {yourName}, for the rest, they shall default to the 'uncontrolled clients' bucket". The client with the specific identifier be a 'controlled' one.
I think you can remove the comment within braces, it use to make sense in a previous iteration. I

Reply:
Done! Thanks!